tools/execute-selenium-tests

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -ex
+
+CURRENT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_HOME="$CURRENT_PATH/.."
+FOXBOX_BINARY="$PROJECT_HOME/target/debug/foxbox"
+
+DEBUG_PORT=${BOX_PORT:-3331}
+DEBUG_PROFILE="$HOME/.local/share/foxbox-debug"
+
+# clean up residual profile
+rm -rf ${DEBUG_PROFILE}
+
+# build and launch foxbox daemon
+pushd ${PROJECT_HOME}
+cargo build
+$FOXBOX_BINARY -p ${DEBUG_PORT} --profile ${DEBUG_PROFILE} &
+FOXBOX_PID=$!
+while ! nc -z localhost ${DEBUG_PORT}; do sleep 3; done
+popd
+
+# run tests
+BOX_PORT=${DEBUG_PORT} ${PROJECT_HOME}/node_modules/.bin/mocha ${PROJECT_HOME}/test/selenium/*_test.js
+
+# clean up foxbox process
+kill $FOXBOX_PID
```

Comment:
I was thinking about keeping it, in case someone wanted to run Link against it and do some manual debugging of what happened.